app/google-services.json

```diff
@@ -64,14 +62,7 @@
       ],
       "services": {
         "analytics_service": {
-          "status": 2,
-          "analytics_property": {
-            "tracking_id": "UA-42015512-1"
```

Comment:
It is generated by firebase. This is the solution suggested here http://stackoverflow.com/a/37549781/243599 to use Firebase alongside GA.